metabase/metabase
# Add semantic type icon/name to dimension info popovers

This PR replaces the `DimensionLabel` found in the `DimensionInfoPopover` with `DimensionSemanticTypeLabel`. Previously, `DimensionLabel` showed the name of the dimension; now, `DimensionSemanticTypeLabel` shows the semantic type of a dimension, if one exists.

<img width="328" alt="Screen Shot 2022-01-17 at 4 48 17 PM" src="https://user-images.githubusercontent.com/13057258/149852232-ac91bc00-9c72-453a-a949-e3a666cef175.png">
<img width="350" alt="Screen Shot 2022-01-17 at 4 48 09 PM" src="https://user-images.githubusercontent.com/13057258/149852235-f99d6098-14bb-4b0f-a626-7a16176bfe7e.png">

**Testing**
1. Navigate to the Products table
2. Hover over headers of the table to view the dimension info popover -- you should now see a label for the associated field's semantic type
3. Go to admin --> data model --> products table and update the semantic type of some fields.
4. Return to the Products table to see the updated labels in the dimension info popover
